tests/std/tests/P0896R4_views_take/test.cpp

```diff
@@ -510,6 +511,13 @@ constexpr void output_range_test() {
     }
 }
 
+void test_VSO1397309() {

```

Comment:
This should be `test_DevCom_1397309` (as VSO is the internal database with a different numbering scheme).